{"uuid":"33791828-7abc-408f-b4df-1099d4093a24","historyId":"7055fa4aafb9d7f3e8b9526dd189bf9e","fullName":"qa.fakestore.tests.AuthenticationTests.testInvalidLogin","labels":[{"name":"package","value":"qa.fakestore.tests.AuthenticationTests"},{"name":"testClass","value":"qa.fakestore.tests.AuthenticationTests"},{"name":"testMethod","value":"testInvalidLogin"},{"name":"parentSuite","value":"FakeStoreAPITestSuite"},{"name":"suite","value":"API Tests"},{"name":"subSuite","value":"qa.fakestore.tests.AuthenticationTests"},{"name":"host","value":"Burak"},{"name":"thread","value":"26184@Burak.TestNG-test=API Tests-2(37)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Invalid User Login"},{"name":"epic","value":"Fake Store API Testing"},{"name":"feature","value":"Authentication"},{"name":"severity","value":"critical"}],"links":[],"name":"testInvalidLogin","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Status code should be 200 expected [200] but found [401]","trace":"java.lang.AssertionError: Status code should be 200 expected [200] but found [401]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1419)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1383)\r\n\tat qa.fakestore.tests.AuthenticationTests.testInvalidLogin(AuthenticationTests.java:65)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\r\n"},"stage":"finished","description":"Verify that login fails with invalid credentials","steps":[],"attachments":[],"parameters":[],"start":1755374775068,"stop":1755374775699}